algorithms/src/polycommit/test_templates.rs

```diff
@@ -283,8 +283,7 @@ where
                 None
             };
 
-            let hiding_bound =
-                if num_points_in_query_set >= degree { Some(degree) } else { Some(num_points_in_query_set) };
+            let hiding_bound = Some(1);
```

Comment:
Only to confirm: is this the right value to check in this test?

Reply:
Yes; we actually do not have the extra powers for satisfying the old hiding_bound, so we have to change the hiding_bound to `Some(1)`